tutorials/examples/cov2d_experiment

```diff
@@ -0,0 +1,52 @@
+"""
+This script illustrates denoising 2D images using batched Cov2D class
+from experime dataset and outputing to mrcs file.
+"""
+
+import logging
+
+from aspire.basis.ffb_2d import FFBBasis2D
+from aspire.denoising.denoiser_cov2d import DenoiserCov2D
+from aspire.estimation.noise import AnisotropicNoiseEstimator
+from aspire.source.relion import RelionSource
+
+logger = logging.getLogger('aspire')
```

Comment:
For a script that you expect to be run by the user, I think you can reduce the cognitive load by just using `print()` statements. A user-facing script doesn't need to interact with aspire's internal logging mechanism.

Reply:
Use `print()` in new PR.